web-ui/src/main/resources/catalog/templates/editor/editor.html

```diff
@@ -1,7 +1,10 @@
 <div data-ng-hide="metadataFound">
-  <div class="alert alert-warning" 
+  <div data-ng-show="!metadataLocked" class="alert alert-warning" 
     data-translate="" 
     data-translate-values="{ id: '{{metadataNotFoundId}}'}">metadataNotFound</div>
+    
+  <div data-ng-show="metadataLocked" class="alert alert-warning" 
```

Comment:
It has to be because Github is slow at updating the code. I pushed a change that allows it: https://github.com/geonetwork/core-geonetwork/commit/5687ad04c6fc917ad67dde3274c3c4714e8915d8